quicktests/overlaying/tests/realistic/stocks.js

```diff
@@ -69,12 +69,14 @@ function run(svg, data, Plottable) {
                                   .addDataset(aaplSource)
                                   .x(function(d) { return d.Date; }, xScale)
                                   .y(function(d) { return d["Adj Close"]; }, yScaleAAPL)
+                                  .interpolator("cardinal")
                                   .attr("stroke", function(d, index, dataset) { return dataset.metadata().name; }, colorScale);
           lineAAPL.autorangeMode("y");
           var lineGOOG = new Plottable.Plots.Line().animated(true)
                                   .addDataset(googSource)
                                   .x(function(d) { return d.Date; }, xScale)
                                   .y(function(d) { return d["Adj Close"]; }, yScaleGOOG)
+                                  .interpolator("cardinal")
```

Comment:
Interesting that you chose to use `interpolator("cardinal")` here. I would think that this test is better for testing `autorangeSmooth()`